swagger.yml

```diff
@@ -3,6 +3,25 @@ info:
   version: 0.11.4
   title: Netlify's API definition
   termsOfService: https://www.netlify.com/legal/terms-of-use/
+  description: >-
+    Netlify is a hosting service for the programmable web. It understands your
+    documents and provides an API to handle atomic deploys of websites, manage
+    form submissions, inject JavaScript snippets, and much more. This is a
+    REST-style API that uses JSON for serialization and OAuth 2 for
+    authentication.
+
+
+    This document describes common usage in some detail. There is also an
+    OpenAPI reference for the Netlify API that you can explore. Visit our
```

Comment:
The "OpenAPI reference" mentioned here is this very page! I would reverse those two descriptions. In other words, explain that this site is a reference, and for more detailed instructions for common uses, visit netlify.com/docs/api.

Reply:
Fixed in f126c128ec640331844f497f6c9ad6967b631d84